packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -806,7 +806,7 @@ export interface CallAdapterDeviceManagement {
    *
    * @public
    */
-  askDevicePermission(constrain: PermissionConstraints): Promise<void>;
+  askDevicePermission(constrain: PermissionConstraints): Promise<DeviceAccess>;
```

Comment:
is it not a breaking change to go from `Promise<void>` to `Promise<something>`? does this follow that this is an additive change?

Reply:
Another good question, it isn't purely on how the expectation of CallAdapter is used.
So if someone implemented their own CallAdapter, it would be a breaking change -- but that would also mean we could never add new functions to this interface.
But if someone is using an adapter then this is not breaking, i.e. their code would be:
```
adapter.askDevicePermission()
```
Which would still work as they don't need to action on the return result

Reply:
Let's hope no one implemented their own CallAdapter.askDevicePermission 😅